getting-started/kubearmor_visibility.md

```diff
@@ -0,0 +1,128 @@
+# KubeArmor Visibility
+
+KubeArmor currently supports enabling visibility for containers/hosts
+Visibility for containers is enabled by default and can be accesed using `karmor` tool
+
+
```

Comment:
We can explain what kind of visibility information would be available. For e.g.,
1. Process
2. Files
3. Networks

Sample screenshots using [collapsible sections](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab).